IFC4x3/Properties/a/AlarmInformation_30Y_YD1x1DY8_4MwqYT1kM/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AlarmInformation_30Y_YD1x1DY8_4MwqYT1kM" Name="AlarmInformation" UniqueId="c08be88d-07b0-4d88-8f84-5bad22741b96" PrimaryDataType="IfcText" />
```

Comment:
Could this be FailureDescription or FailureInformation, I find AlarmInformation to be misleading, as an Alarm is a particular type of instance.